tools/PI/DevHome.PI/Controls/ProgressTextRing.xaml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.PI.Controls.ProgressTextRing"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.PI.Controls"
```

Comment:
I don't think you need to define local here.

Reply:
Deleted.